user_guide/data_management_access_permissions_basic.rst

```diff
@@ -0,0 +1,112 @@
+    have this right, regardless of the settings defined for other roles of this user.
+    
+
+Actions with Entities    
+^^^^^^^^^^^^^^^^^^^^^
+
+There is a number of actions that can be permitted for each combination of the role, organization/unit and entity.
+For example, it can be defined that any user with the role "Sales Representative" who belongs to the business unit 
+"California" can create and edit records of Customers, if this records were created by this user or by any other user 
+from this business unit, and can only view (but not edit) similar records created by users from other business units 
+(e.g. "Texas" or "Florida"). At the same time, for users with another role, for example "System Administrator", this 
+records may be even not seen.
+
+The following actions can be permitted for each role, organization/unit and entity:
+  
+  
+- **View**: If, for a specific entity, the action is not available to a user, the user won't see the records 
+  :ref:`Grids <user-guide-ui-components-grids>` nor the :ref:`View pages <user-guide-ui-components-view-pages>` 
+  of this entity records.
+  
+- **Create**: If, for a specific entity, the action is not available to a user, the user won't be able to create new 
+  entity records.
+
+- **Edit**: If, for a specific entity, the action is not available to a user, the user won't be able to edit the entity 
+  records.
+
+- **Delete**: If, for a specific entity, the action is not available to a user, the user won't be able to delete the
+  entity records.
+  
+- **Assign**: If, for a specific entity, the action is not available to a user, the user won't be able to change the owner 
+  of the entity records.
+
+
+Working in System Organizations
+-------------------------------
+
+OroCRM Enterprise edition support the use of :ref:`multiple organizations <user-ee-multi-org>`. When this functionality
+is used, there can be any number of non-system and one system organization created. The users, assigned to the system 
+organization can perform an action for the entity records in any organization within the system, if they are assigned a 
+role, for which the relevant permission is set to *"System"*.
+
+      |
+  
+.. image:: ./img/multi_org/multi_org_permission.png
+
+|
+
+For example, to enable your support desk to view details of all the customers), there should be:
+
+- A system organization (e.g. "Main Office")
+
+- A role created(e.g. "Support")
+
+- Permission to view the customer entity defined within the "Main Office" set to "System"
+
+- Users of the support desk must have the role "Support" and be assigned to the "Main Office" organization. 
+
+
+Conclusion
+----------
+
+Your screen may not always look as in the guides, as some capabilities may not be available to you. Please address your 
+system administrator, if you feel that some functionality or capability is required for your work. 
```

Comment:
Edit: Your screen may not always look like the images in this guide, as some capabilities may not be available to you. Please contact your system administrator if you require a specific functionality or capability for your work.